app_build_suite/build_steps/helm.py

```diff
@@ -382,6 +403,72 @@ def pre_run(self, config: argparse.Namespace) -> None:
                 ):
                     raise ValidationError(self.name, f"Value of '{option}' is not a correct boolean.")
 
+    def additional_annotations(self, chart_file_name: str, catalog_name: str) -> Dict[str, Any]:
+        catalog_base_url = f"{self._value_annotation_catalog_base_url}{catalog_name}/{chart_file_name}-meta/"
+        annotations = {}
+        # TODO check if these exist
```

Comment:
Yes, we definetely have to check if source files exist. I think this should be checked in `pre_run`, as it's a fast check and `pre_run` is meant to allow the build to fail fast if it will fail anyway later.